src/ButtonCatalog.js

```diff
@@ -16,11 +17,16 @@ const ButtonCatalog = () => {
       docsLink='https://material.io/components/web/catalog/buttons/'
       sourceLink='https://github.com/material-components/material-components-web/tree/master/packages/mdc-button'
       demos={<ButtonDemos />}
+      initialConfig={ButtonConfig}
+      {...props}
```

Comment:
What is props being used for in the places it's added?

Reply:
It's so that the props are passed into the `ComponentCatalogPanel`. The props have the location information that we're using to store state, so they need to be passed all the way down to the Hero... components.